repair_quotation_manual_sync/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Repair Quotation Manual Sync",
+    "summary": """
+        Synchronize Repair Orders with Sales Quotations
```

Comment:
```suggestion
        Manually Synchronize Repair Orders with their Quotations
```
Here I believe the concept of manual is the key one.